pkg/osd_cluster_upgrader/upgrader.go

```diff
@@ -395,10 +398,12 @@ func performUpgradeVerification(c client.Client, metricsClient metrics.Metrics,
 	totalDS := 0
 	for _, daemonSet := range daemonSetList.Items {
 		for _, namespacePrefix := range namespacePrefixesToCheck {
-			if strings.HasPrefix(daemonSet.Namespace, namespacePrefix) {
-				totalDS = totalDS + 1
-				if daemonSet.Status.DesiredNumberScheduled == daemonSet.Status.NumberReady {
-					readyDS = readyDS + 1
+			for _, ignoredNS := range namespaceToIgnore {
+				if strings.HasPrefix(daemonSet.Namespace, namespacePrefix) && daemonSet.Namespace != ignoredNS {
```

Comment:
This is fine and should work, but I think it might lead to an inflated count of the number of daemonsets when reported in the subsequent log. (it'll be 'actual number of daemonsets' * 'number of ignored namespaces'). 
Just a very small thing that only really matters when it comes to the resulting log. I would be fine if it's not dealt with.